course/views.py

```diff
@@ -1041,8 +1041,9 @@ def monitor_task(request, task_id):
             progress_percent = 100 * (current / total)
 
         progress_statement = (
-                _("%d out of %d items processed.")
-                % (current, total))
+                _("%(current)d out of %(total)d items processed.")
```

Comment:
Just for your reference, if more than 1 string is passed, we need to use this ``dict`` format to map them, or else ``makemessages`` will raise errors.